src/Sulu/Bundle/MediaBundle/Resources/config/doctrine/FileVersion.orm.xml

```diff
@@ -8,7 +8,6 @@
             <index columns="mimeType"/>
```

Comment:
this need to be set also to 191 did set it to 255 to test travis with utf8mb4 but travis seems to be not failing with utf8mb4. Seems like travis mysql runs on innodb barracuda driver and not the default storage of mysql.